test_crates/struct_now_doc_hidden/old/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+struct StayingKnown;
```

Comment:
Did you intentionally leave this struct private in both `old` and `new`? It was never part of the public API in the first place.

Also btw we already have a test configuration where we lint `old` vs itself, and `new` vs itself, to make sure that we never raise lints when nothing changed.

Reply:
> Did you intentionally leave this struct private in both old and new? It was never part of the public API in the first place.

No I didn't, it's been removed.

> Also btw we already have a test configuration where we lint old vs itself, and new vs itself, to make sure that we never raise lints when nothing changed.

Oh cool!